Mathlib/Analysis/Complex/Angle.lean

```diff
@@ -0,0 +1,65 @@
+/-
+Copyright (c) 2024 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Analysis.SpecialFunctions.Complex.Arg
+import Mathlib.Geometry.Euclidean.Angle.Unoriented.Basic
+
+/-!
+# Angle between complex numbers
+
+This file relates the Euclidean geometric notion of angle between complex numbers to the argument of
+their quotient.
+
+## Tags
+
+arc-length, arc-distance
+-/
+
+open InnerProductGeometry
+open scoped Real
+
+namespace Complex
+variable {a x y : ℂ}
+
+/-- The angle between two non-zero complex numbers is the absolute value of the argument of their
+quotient.
+
+Note that this does not hold when `x` or `y` is `0` as the LHS is `π / 2` while the RHS is `0`. -/
+lemma angle_eq_abs_arg (hx : x ≠ 0) (hy : y ≠ 0) : angle x y = |(x / y).arg| := by
```

Comment:
Do you think it makes sense to prove things about `oangle` too? Fine to leave to a later PR.

Reply:
Added a todo